legalhtml/README.md

```diff
@@ -0,0 +1,13 @@
+# /legalhtml/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for LegalHTML resources.
+
+## Contact
+This space is administered by:
+
+**Manuel Fiorelli**
+*Research Fellow*
+[University of Rome Tor Vergata](https://web.uniroma2.it/)
+Roma, Italy
+<manuel.fiorelli@uniroma2.it>
+GitHub: [manuelfiorelli](https://github.com/manuelfiorelli)
+ORCID: [0000-0001-7079-8941](https://orcid.org/0000-0001-7079-8941)
```

Comment:
If you want to preserve these line breaks, as I believe you do, each line must end with two space characters, as suggested here --
```suggestion
**Manuel Fiorelli**  
*Research Fellow*  
[University of Rome Tor Vergata](https://web.uniroma2.it/)  
Roma, Italy  
<manuel.fiorelli@uniroma2.it>  
GitHub: [manuelfiorelli](https://github.com/manuelfiorelli)  
ORCID: [0000-0001-7079-8941](https://orcid.org/0000-0001-7079-8941)
```

Reply:
Hi @TallTed you were right about my intention to preserve line breaks; therefore, I committed your suggested change.